README.md

```diff
@@ -54,6 +54,13 @@ The PMEM-CSI driver follows the [CSI
 specification](https://github.com/container-storage-interface/spec) by
 listening for API requests and provisioning volumes accordingly.
 
+The [v0.6.0
+release](https://github.com/intel/pmem-csi/releases/tag/v0.6.0) is the
+latest feature release and is [getting
+updated](./DEVELOPMENT.md#release-management) with newer base images
+regularly and with bug fixes as needed, so the latest [v0.6.x
```

Comment:
There is a convenient link https://github.com/intel/pmem-csi/releases/latest to get to latest release, should we mention it here? It's not so good to point to latest link while we explicitly list releases by number though, but latest link exists and is helpful for user.

Reply:
The intention is to document what the latest release is, not just describe how to find it.

How would you add the link (full sentence)?

Reply:
yeah, it may not be so good idea after all to duplicate link to latest release, it;'s not that difficult to locate by anyone who is interested in releases. Let's not add it.